src/linux/ptrace_dumper.rs

```diff
@@ -374,7 +363,7 @@ impl PtraceDumper {
         // the bitfield length is 2^test_bits long.
         let test_bits = 11;
         // byte length of the corresponding array.
-        let array_size = 1 << (test_bits - 3);
+        let array_size: u64 = 1 << (test_bits - 3);
```

Comment:
I think this should be an `usize`, it would remove several redundant casts and ensure we don't have an array_size that's larger than the pointer size on 32-bits hosts.

Reply:
Sounds good to me. To be honest, I'm not sure why I had changed it to an explicit type, previously I believe it was inferred as `usize` based on the usage as an index.